testnet/stacks-node/src/tests/epoch_205.rs

```diff
@@ -913,3 +916,292 @@ fn test_cost_limit_switch_version205() {
+                       0xd0 0xd1 0xd2 0xd3 0xd4 0xd5 0xd6 0xd7 0xd8 0xd9 0xda 0xdb 0xdc 0xdd 0xde 0xdf
+                       0xe0 0xe1 0xe2 0xe3 0xe4 0xe5 0xe6 0xe7 0xe8 0xe9 0xea 0xeb 0xec 0xed 0xee 0xef
+                       0xf0 0xf1 0xf2 0xf3 0xf4 0xf5 0xf6 0xf7 0xf8 0xf9 0xfa 0xfb 0xfc 0xfd 0xfe 0xff
+                    ))
+                    (define-private (crash-me-folder (input (buff 1)) (ctr uint))
+                        (begin
+                            (unwrap-panic (index-of BUFF_TO_BYTE input))
+                            (unwrap-panic (index-of BUFF_TO_BYTE input))
+                            (unwrap-panic (index-of BUFF_TO_BYTE input))
+                            (unwrap-panic (index-of BUFF_TO_BYTE input))
+                            (unwrap-panic (index-of BUFF_TO_BYTE input))
+                            (unwrap-panic (index-of BUFF_TO_BYTE input))
+                            (unwrap-panic (index-of BUFF_TO_BYTE input))
+                            (unwrap-panic (index-of BUFF_TO_BYTE input))
+                            (+ u1 ctr)
+                        )
+                    )
+                    (define-public (crash-me (name (string-ascii 128)))
+                        (begin
+                            (fold crash-me-folder BUFF_TO_BYTE u0)
+                            (print name)
+                            (ok u0)
+                        )
+                    )
+                    (begin
+                        (crash-me \"{}\"))
+                    ",
+                    &format!("large-contract-{}", &ix)
+                )
+            )
+        })
+        .collect();
+
+    let (mut conf, miner_account) = neon_integration_test_conf();
+
+    for spender_addr in spender_addrs.iter() {
+        conf.initial_balances.push(InitialBalance {
+            address: spender_addr.clone(),
+            amount: 10492300000,
+        });
+    }
+
+    conf.node.mine_microblocks = true;
+    conf.node.wait_time_for_microblocks = 1000;
+    conf.node.microblock_frequency = 1000;
+    conf.node.max_microblocks = 65536;
+    conf.burnchain.max_rbf = 1000000;
+
+    conf.miner.min_tx_fee = 1;
+    conf.miner.first_attempt_time_ms = i64::max_value() as u64;
+    conf.miner.subsequent_attempt_time_ms = i64::max_value() as u64;
+
+    conf.burnchain.epochs = Some(vec![
+        StacksEpoch {
+            epoch_id: StacksEpochId::Epoch20,
+            start_height: 0,
+            end_height: 208,
+            block_limit: ExecutionCost {
+                write_length: 15000000,
+                write_count: 7750,
+                read_length: 100000000,
+                read_count: 7750,
+                runtime: 5000000000,
+            },
+            network_epoch: PEER_VERSION_EPOCH_2_0,
+        },
+        StacksEpoch {
+            epoch_id: StacksEpochId::Epoch2_05,
+            start_height: 208,
+            end_height: 9223372036854775807,
+            block_limit: ExecutionCost {
```

Comment:
Why are you doubling the default limits in 2.05?

Reply:
I can just double the read/write counts and leave everything else untouched